ai-gen-tools/earthquake-shakemap-urls/scripts/build_interactive.py

```
@@ -0,0 +1,156 @@
+import os
+import sys
+import urllib.request
+
+SKILL_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+TEMPLATE_PATH = os.path.join(SKILL_DIR, "assets", "template_interactive.html")
+TURF_JS_PATH = os.path.join(SKILL_DIR, "assets", "turf.min.js")
+BASEMAP_PATH = os.path.join(SKILL_DIR, "assets", "ne_countries_110m.min.geojson")
+
+
+def fetch_json(url):
+    req = urllib.request.Request(url, headers={"User-Agent": "earthquake-shakemap-urls-skill"})
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)